js/app.js

```diff
@@ -0,0 +1,47 @@
+$(function(){
+
+
+  //listen for a click event on the "update" button, when clicked, execute function "read"
+  $('#submit-btn').click(read)
+
+
+
+  //define function "read"
+  function read() {
+  //read the value in the city-type input box and store it as a variable  
+    var city = $('#city-type').val();
```

Comment:
@zdburnett consider making the inputs case the same as your desired validation to cover form validation